### DISCUSSION TITLE
How to open or trigger the cards + grid view?

### DISCUSSION THREAD

**Discussion by incertoe** [levito/tt-rss-feedly-theme#106] [Q&A] [answered]
I have installed the theme and it looks great so far!! I'm curious how I'm supposed to be able to see the grid view in Tiny Tiny RSS that is viewable in the screenshots?

**Comment by levito** [comment, marked as answer]
Hi, in preferences, 1st tab, 2nd accordion pane (also called "Preferences") in the section "Articles", enable "Combined mode" and "Grid view", and disable "Always expand articles". This enables the expanded, large grid view. For what I called "cards view" (not available in the stock theme), enable "Always expand articles".

**Reply by whitezo** [reply]
Hi, for me, only with always expand I get grid view, but only on the stock theme, and maybe two per line on a bigger than 13" screen. On this theme however I can't produce anything like that. If I turn off always expand I simply see a list view (even though grid is checked). This with having updated both TTRSS and the theme...

**Reply by levito** [reply]
Could you please check if you maybe have custom styles defined? Also, please check if there is an older version of this theme in `themes` while you updated the one in `themes.local`. It might also be a cache issue.

**Reply by whitezo** [reply]
Thanks for the reply. I've deleted and re-copied the theme files (fresh pull from the theme and ttrss) but still the same. In fact, I had to add custom styles earlier (which I've removed during the test) to fix various issues: material icons not visible (text placeholders everywhere), spinners/loading indicators very large in various areas:

```
.icon-three-dots, .loading-indicator { max-width: 20px; }
.loadingNode { max-width: 16px; }
@font-face {
  font-family: 'Material Icons';
 ...
}
.material-icons {
    font-family: 'Material Icons';
   ...
}
```

**Reply by levito** [reply]
If you needed to make adjustments like that it means that there is a version mismatch. Have you checked the `themes` directory of your tt-rss installation? I believe there might be an old version of this theme there which takes precedence over the one in `themes.local`.

**Reply by whitezo** [reply]
Wow thanks, you were right! There was an older version in the regular `themes` dir as well, sorry for the inconvenience :).
It's been a while since I've last updated my setup, so I had to get familiar with it again...
